src/app/ui/export_file_window.cpp

```diff
@@ -192,8 +192,13 @@ void ExportFileWindow::setAniDir(const doc::AniDir aniDir)
 
 void ExportFileWindow::setOutputFilename(const std::string& pathAndFilename)
 {
-  m_outputPath = base::get_file_path(pathAndFilename);
-  m_outputFilename = base::get_file_name(pathAndFilename);
+  if (base::get_file_path(m_doc->filename()).empty()) {
+    m_outputPath = base::get_file_path(pathAndFilename);
+    m_outputFilename = base::get_file_name(pathAndFilename);
+  } else {
+    m_outputPath = base::get_file_path(m_doc->filename());
+    m_outputFilename = base::get_relative_path(pathAndFilename, base::get_file_path(m_doc->filename()));
+  }
```

Comment:
@dacap , when we save a file (not export it), we cannot pass a dynamic placeholder like `{tag}`, so the relative path works as spected.

But, when we only export the file that is not saved, we can't get the path of the `.aseprite` file in `m_doc->filename()` and, therefore, it will just split the `pathAndFilename` in the old way.

For instance, the path `out/{tag}/test.png` would only show `test.png` in the input, causing the same bug as before.

The problem is that `m_docPref.saveCopy` has the full  `out/{tag}/test.png` path and we cannot get the relative path to an inexistent file.

In this case, what should we do? I thought about showing the absolute path but I'm not sure if this is the best approach.

Reply:
@gui-marc I've just tested with your laf changes and it looks good so far :+1: At least it fixes the original issue for most cases (when the sprite is saved/has a filename associated).

We can merge this and I can think about a future fix for the other cases (there is a general UI problem to show the full path, or to show the real output folder when pressing the "..." dots, related to #4356).

Reply:
I forgot, you'll need to create a PR for the laf project with your get-relative-path branch.

Reply:
@dacap, ok, I was waiting for your reply here before creating the PR in the laf repo. I'm creating it now

Reply:
@dacap I've just oppened the PR: https://github.com/aseprite/laf/pull/96